metadata/manifest.yml

```diff
@@ -104,13 +102,6 @@ Endpoints:
     Private-Port-Name: CLUSTER_PORT
     Private-Port:      7600
     Public-Port-Name:  CLUSTER_PROXY_PORT
-  - Private-IP-Name:   IP
-    Private-Port-Name: MESSAGING_PORT
-    Private-Port:      5445
-    Public-Port-Name:  MESSAGING_PROXY_PORT
-  - Private-IP-Name:   IP
-    Private-Port-Name: MESSAGING_THROUGHPUT_PORT
-    Private-Port:      5455
```

Comment:
@matzew Just checked the WildFly cartridge's manifest file, it no longer includes the "MESSAGING_" variables here, so I think you should be able to remove it as well.